cmds/contest-generator/contest.go.template

```diff
@@ -0,0 +1,78 @@
+{{/* This file is the template source. The following comment obviously doesn't apply here */ -}}
+// This is a generated file, edits should be made in the corresponding source
+// file, and this file regenerated using
+//   `contest-generator --from core-plugins.txt`.
```

Comment:
this is a template that will be used with other configs, so the command will be wrong for uses other than core-plugins.
i think the template should have a variable, like {{ .CommandLine }}

Reply:
I thought of that comment just as an example, not as the actual command. Maybe I can replace it with "--from your-plugins-configuration.yml" to be more clear?

Reply:
changed, let me know if that's OK

Reply:
ah I see. The generic name is also incorrect for `cmds/contest/main.go`

Reply:
well, now it won't match anyone's config :)
i was thinking of something along the lines of:

```
// Generated with:
//  { .CommandLine }}
```
with
```
  CommandLine: strings.Join(or.Args, " ")
```

or some such

Reply:
done, with filename passed to the templaet

Reply:
@rojer do you think we need the full command line, or a basic `contest-generator --from $filename` is enough? For example, if I run it with `go run` to regenerate cmds/contest/main.go,  it will look like
```
/tmp/somewhere/contest-generator -o ../contest/main.go
```
which is less useful than
```
contest-generator --from core-plugins.yml
```
(the plugins config file is not part of the command line, it's taken from the defaults)